deploy/ssh.sh

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env sh
+
+# Script to deploy certificates to remote server by SSH
+# Note that SSH must be able to login to remote host without a password...
+# SSH Keys must have been exchanged with the remote host.  Validate and
+# test that you can login to USER@SERVER from the host running acme.sh before
+# using this script.
+#
+# The following variables exported from environment will be used.
+# If not set then values previously saved in domain.conf file are used.
+#
+# Only a username is required.  All others are optional.
+#
+# The following examples are for QNAP NAS running QTS 4.2 
+# export ACME_DEPLOY_SSH_USER="admin"
+# export ACME_DEPLOY_SSH_SERVER="qnap"
+# export ACME_DEPLOY_SSH_PORT="22"
+# export ACME_DEPLOY_SSH_SERVICE_STOP=""
+# export ACME_DEPLOY_SSH_KEYFILE="/etc/stunnel/stunnel.pem"
+# export ACME_DEPLOY_SSH_CERTFILE="/etc/stunnel/stunnel.pem"
+# export ACME_DEPLOY_SSH_CAFILE="/etc/stunnel/uca.pem"
+# export ACME_DEPLOY_SSH_FULLCHAIN=""
+# export ACME_DEPLOY_SSH_REMOTE_CMD="/etc/init.d/stunnel.sh restart"
+# export ACME_DEPLOY_SSH_SERVICE_START=""
```

Comment:
I am open to suggestions on how to simplify.  It is possible to chain multiple remote commands into a single variable.... I have tested with cmd1 && cmd2 and it worked.  In the case of Unifi Controller I would need to openssl && keytool && service start so three commands would be chained.

The openssl btw is to create a pkcs12, something that acme.sh can do but then that is a binary file which is not easy to send over to the remote server by ssh, so I generate the pkcs12 at the remote end.